satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -102,7 +102,7 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 							zap.Error(err))
 						log.Debug("register: ", zap.String("rawSecret", secretInput))
 
-						return nil, err
+						return nil, HandleError(err)
```

Comment:
we can add middleware to handle error in one place

Reply:
Where?